server/tinylicious/src/routes/ordering/documents.ts

```diff
@@ -72,5 +81,33 @@ export function create(storage: IDocumentStorage): Router {
 		);
 	});
 
+	/**
+	 * Passes on content to all clients in a collaboration session happening on the document via means of signal.
+	 */
+	router.post("/:tenantId/:id/broadcast-signal", (request, response) => {
+		const tenantId = getParam(request.params, "tenantId");
+		const documentId = getParam(request.params, "id");
+		const signalContent = getParam(request.body, "signalContent");
```

Comment:
Nit: let's add a comment to clarify that the signal is a blackbox to this endpoint, and that's why we don't do any validations on it.